EchoBot.cs

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+//
+// Generated with Bot Builder V4 SDK Template for Visual Studio EchoBot v4.6.2
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+using Microsoft.Bot.Builder.Teams;
+using Microsoft.Bot.Schema.Teams;
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Microsoft.IdentityModel.JsonWebTokens;
+using Microsoft.IdentityModel.Tokens;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using AdaptiveCards;
+//using System.Security.Cryptography.HashAlgorithm;
+
+namespace HelloWorldBot.Bots
+{
+    public class EchoBot : TeamsActivityHandler
+    {
+        protected override async Task<MessagingExtensionResponse> OnTeamsAppBasedLinkQueryAsync(ITurnContext<IInvokeActivity> turnContext, AppBasedLinkQuery query, CancellationToken cancellationToken)
```

Comment:
Can we break this line after `IInvokeActivity> turnContext`  to a new line for better visibility